patches/net/minecraft/client/gui/Gui.java.patch

```diff
@@ -386,6 +386,16 @@
                  }
              }
          }
+@@ -1037,7 +_,8 @@
+         int l = (p_282069_.guiHeight() - j) / 2;
+         int i1 = k + i;
+         int j1 = l + j;
+-        p_282069_.blit(RenderPipelines.GUI_TEXTURED, SPYGLASS_SCOPE_LOCATION, k, l, 0.0F, 0.0F, i, j, i, j);
++        ItemStack useItem = this.minecraft.player.getUseItem();
++        p_282069_.blit(RenderPipelines.GUI_TEXTURED, net.neoforged.neoforge.client.extensions.common.IClientItemExtensions.of(useItem).getScopingTexture(useItem), k, l, 0.0F, 0.0F, i, j, i, j);
```

Comment:
Prefer moving the `IClientItemExtensions` into a local variable for readability (use `var` to avoid typing out the variable's type).